src/MCPClient/lib/clientScripts/moveToBacklog.py

```diff
@@ -6,11 +6,13 @@
 import sys
 
 # archivematicaCommon
+from custom_handlers import GroupWriteRotatingFileHandler
 import storageService as storage_service
 
-logger = logging.getLogger(__name__)
-logging.basicConfig(filename="/tmp/archivematica.log",
-    level=logging.INFO)
+logger = logging.getLogger('archivematica.mcp.client')
+logger.addHandler(GroupWriteRotatingFileHandler("/var/log/archivematica/dashboard/dashboard.log",
```

Comment:
Yeah, maybe these should log to `/var/log/archivematica/MCPClient/{scriptname}.log`?